source/documentation/common/security/proc-Applying_the_DISA_STIG_Security_Profile_RHV.adoc

```diff
@@ -40,3 +36,13 @@ These packages are already part of the {hypervisor-shortname} image. {hypervisor
 
 When you log into {hypervisor-shortname}, the command line displays the following information.
 image::common/images/security-policy-rhvh-login.png[Login notice]
+
+= Enabling DISA STIG in a self-hosted engine
```

Comment:
@emarcusRH @almusil @sandrobonazzola  
All assemblies and modules must comply with downstream build conventions. The heading level is incorrect and was causing site build errors. I fixed it in https://github.com/oVirt/ovirt-site/pull/2961.